data/episodes/episodes.json

```diff
@@ -0,0 +1,84 @@
+{
+  "episodes": [
```

Comment:
A couple of other data elements that an episode needs:

- youtube URL (this only should display if it exists; we'll add it after the episode is posted to YT)
- date
- start time
- google calendar URL (should only display if the episode is in the past; could get lazy and make it only display if it exists, and we can delete it from the data when we update with the youtube URL, etc)